emcc.py

```diff
@@ -1387,7 +1387,7 @@ def has_c_source(args):
     if shared.Settings.EMBIND:
       forced_stdlibs.append('libembind')
 
-    if not shared.Settings.MINIMAL_RUNTIME:
+    if not shared.Settings.MINIMAL_RUNTIME and not shared.Settings.STANDALONE_WASM:
```

Comment:
Please add a comment here explaining why (`our default JS glue requires malloc for some things, but the minimal runtime glue or not having glue at all avoids that`, or such)

Reply:
Done